src/index.js

```diff
@@ -1,3 +1,6 @@
 import { run } from "./action.js";
+import * as util from "./util.js";
 
-run();
+if (util.shouldRun()) {
+  run();
```

Comment:
let's print a message explaining why we're not running for helping with debugging. let's maybe use annotations? no idea what they do but sounds appropriate https://github.com/actions/toolkit/tree/main/packages/core#annotations

Reply:
Done

Reply:
can't see it

Reply:
https://github.com/nearform/github-action-check-linked-issues/pull/12/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R7